src/test/java/seedu/recipe/logic/commands/FindCommandTest.java

```diff
@@ -94,8 +99,10 @@ public void execute_namePropertyMultipleKeywords_multipleRecipesFound() {
     public void execute_tagPropertyMultipleKeywords_multipleRecipesFound() {
```

Comment:
you can use `three` or `multiple` here as long as the previous one is `two`, so this is up to you.